src/Tasks/MigrationTaskStep2.php

```diff
@@ -0,0 +1,321 @@
+     *  - `ID`: ID for the relationship
+     *
+     * @var string
+     */
+    private static $has_one_sql_template = <<<SQL
+SELECT
+    "ft"."FilePublicID",
+    "lt"."Focus",
+    CASE
+        WHEN "lt"."Caption" IS NOT NULL THEN "lt"."Caption"
+        ELSE "ft"."Caption"
+    END AS "Caption",
+    CASE
+        WHEN "lt"."AltText" IS NOT NULL THEN "lt"."AltText"
+        ELSE "ft"."AltText"
+    END AS "AltText"
+FROM "{LinkTable}" As lt
+INNER JOIN "{FileTable}" As ft ON "ft"."ID" = "lt"."FileID"
+WHERE "lt"."ID" = (SELECT "{FieldName}ID" FROM "{TableName}" WHERE "ID" = '{ID}')
+SQL;
+
+    /**
+     *
+     * Variables used are:
+     *  - `RelationTableName`:
+     *  - `LinkTable`:
+     *  - `FileTable`:
+     *  - `RootRelationName`:
+     *  - `ID`:
+     *
+     * e.g.
+     * ```
+     * SELECT
+     *     `f`.`FilePublicID`,
+     *     CASE
+     *         WHEN `fl`.`Caption` IS NOT NULL THEN `fl`.`Caption`
+     *         ELSE `f`.`Caption`
+     *     END AS `Caption`
+     *     CASE
+     *         WHEN `fl`.`AltText` IS NOT NULL THEN `fl`.`AltText`
+     *         ELSE `f`.`AltText`
+     *     END AS `AltText`
+     *     `fl`.`Focus`
+     * FROM `ProductPage_Images` AS l
+     * INNER JOIN `CloudinaryImageLink` AS fl ON `fl`.`ID` = `l`.`CloudinaryImageLinkID`
+     * INNER JOIN `File` AS f ON `f`.`ID` = `fl`.`FileID`
+     * WHERE
+     * 	`l`.`ProductPageID` = 140
+     * ;
+     * ```
+     *
+     * @var string
+     */
+    private static $many_many_sql_template = <<<SQL
+SELECT
+    "f"."FilePublicID",
+    CASE
+        WHEN "fl"."Caption" IS NOT NULL THEN "fl"."Caption"
+        ELSE "f"."Caption"
+    END AS "Caption",
+    CASE
+        WHEN "fl"."AltText" IS NOT NULL THEN "fl"."AltText"
+        ELSE "f"."AltText"
+    END AS "AltText",
+    "fl"."Focus"
+FROM "{RelationTableName}" AS l
+INNER JOIN "{LinkTable}" AS fl ON "fl"."ID" = "l"."{LinkTable}ID"
+INNER JOIN "{FileTable}{VersionSuffix}" AS f ON "f"."ID" = "fl"."{FileTable}ID"
+WHERE
+	"l"."{RootRelationName}ID" = '{ID}'
+;
+SQL;
+
+    private $requestCount = 0;
+
+    public function __construct(Cloudinary $cloudinary, CacheInterface $cache)
+    {
+        $this->cloudinary = $cloudinary;
+        $this->cache = $cache;
+
+        Security::setCurrentUser(DefaultAdminService::singleton()->findOrCreateDefaultAdmin());
+        $this->apiController = Injector::inst()->get('MadeHQ\Cloudinary\Controllers\API');
+    }
+
+    public function run($request)
+    {
+        if (!Director::is_cli()) {
+            echo '<pre>';
+        }
+
+        $dataObjectClasses = ClassInfo::subclassesFor(DataObject::class, false);
+        $schema = DataObject::getSchema();
+
+        /**
+         *
+         */
+        foreach($dataObjectClasses As $className) {
+            DataObject::get($className)->each(function (DataObject $do) use ($schema) {
+                $stages = [
+                    Versioned::DRAFT => '',
+                ];
```

Comment:
If you want to follow the coding standard defined by PSR-2 or PSR-12 then you should have line break after a line of code.

Reply:
Added line break